### PR TITLE
add elevate to delegates

### PR DESCRIPTION
**Pull request by tobiaslins**
I already introduced myself in slack :)

I'm Tobi - 22 years old and from Austria! I love coding (Fullstack JS) and ofc lisk!
I already did some contributions to lisk-js and one small change for lisk-core!
I am currently working on some lisk projects in my spare time:
- Lisk Telegram Bot (@LiskToolBot) 
- Lisk Pending API (For the Bot)
- Lisk Mobile (in work)
- Lisk Delegate (elevate) (in work)
- Lisk Pool (new nodejs based pool; currently planning - want your Feedback if useful)
- Lisk.Supply (Brand of all products listed above) 

And planning to do much more

### CODE REVIEW

File: data/delegates.json [hunk 1]
```diff
@@ -258,5 +258,22 @@
     "poolMember": false,
     "required": false,
     "featured": false
+  },
+  {
+    "delegateName": "elevate",
+    "delegateAddress": "17890508407355636952L",
+    "proposal": "",
```
Comment (4fryn) [inline data/delegates.json]: Hey, thanks for your submission. A proposal is not a must-have, but in general, it's the staging point for everyone to learn about you and yourself and your projects. Also, a forum proposal can be automatically linked to your delegate on the block explorer.

**Comment by 4fryn** [comment]
Test fails unrelated.

File: data/delegates.json (hunk 1 above)
Comment (tobiaslins) [reply data/delegates.json]: hey @4fryn :)

I already added the proposal! And thanks for the tip with the block explorer!

**Comment by 5an1ty** [comment]
We have discussed and you are a solid candidate, welcome to the team!

**Pull request merged**